opal/include/opal/sys/powerpc/atomic.h

```diff
@@ -293,54 +287,6 @@ static inline int64_t opal_atomic_swap_64(opal_atomic_int64_t *addr, int64_t new
 
 #    endif /* OPAL_GCC_INLINE_ASSEMBLY */
 
-#elif (OPAL_ASSEMBLY_ARCH == OPAL_POWERPC32) && OPAL_ASM_SUPPORT_64BIT
-
-#    ifndef ll_low /* GLIBC provides these somewhere, so protect */
-#        define ll_low(x)  *(((unsigned int *) &(x)) + 0)
-#        define ll_high(x) *(((unsigned int *) &(x)) + 1)
-#    endif
-
-#    if OPAL_GCC_INLINE_ASSEMBLY
-
-static inline bool opal_atomic_compare_exchange_strong_64(opal_atomic_int64_t *addr,
```

Comment:
Does this need to be removed? I would think that only the _32 functions would need to be purged.

Reply:
That was for the G5 before the OS was always 64-bit. It shouldn't be needed anymore.

Reply:
Double-checked and this is indeed for the G5. It implemented a compare-exchange 64-bit on PPC32. Since we are deleting PPC32 this is no longer needed. I don't think IBM even supports PPC32 anymore.

Reply:
Ah, I see. I missed the #elif on line 296. LGTM. Thanks!